docs/apidesc.md

```diff
@@ -66,8 +66,8 @@ one exception is NICs, where the label is unique only on a per-node basis.
     project_connect_node        <project_label> <node_label>
     project_detach_node         <project_label> <node_label>
  
-    node_connect_network        <node_label> <nic_label> <network_label>
-    node_detach_network         <node_label> <nic_label>
+    node_connect_network        <node_label> <nic_label> <network_label> [ <channel> ]
+    node_detach_network         <node_label> <nic_label> <network_label>
```

Comment:
Quoting George Silvis, III (2015-05-23 00:44:43)

>    Making it optional could help make it backwards-compatible. Some 
>    ways
>    of doing this are: remove the native VLAN; remove /everything/; remove
>    the native VLAN if it's the only one [as [2]@henn suggested]. I prefer
>    the first---it's most parallel with how we're doing
>    node_connect_network.

I think I agree -- I suppose if a user is blindly ignoring the optional 
field, this works as expected; they'll never set any other channel. I'm 
fine with making it optional.